pkg/api/common/versions.go

```diff
@@ -263,8 +263,8 @@ var AllKubernetesSupportedVersionsAzureStack = map[string]bool{
 	"1.16.10": false,
 	"1.16.11": false,
 	"1.16.13": false,
-	"1.16.14": true,
-	"1.16.15": true,
+	"1.16.14": false,
```

Comment:
map `AllKubernetesWindowsSupportedVersionsAzureStack` (a few lines below) also needs to be updated.